README.md

```diff
@@ -380,14 +380,30 @@ Useful in [Continuous integration (CI)](https://en.wikipedia.org/wiki/Continuous
 #### --cd-version
 
 ```sh
-$ lerna publish --cd-version (patch | major | minor)
+$ lerna publish --cd-version (patch | major | minor | prepatch | premajor | preminor | prerelease)
```

Comment:
What do you think about ordering these to be consistent with `npm version`, which, from `npm help version` is:

```
npm version [<newversion> | major | minor | patch | premajor | preminor | prepatch | prerelease | from-git]
```